src/Apps/Order/Routes/Offer/index.tsx

```diff
@@ -0,0 +1,226 @@
+import { Button, Flex, Sans, Spacer } from "@artsy/palette"
+import { Offer_order } from "__generated__/Offer_order.graphql"
+import { Helper } from "Apps/Order/Components/Helper"
+import { TransactionSummaryFragmentContainer as TransactionSummary } from "Apps/Order/Components/TransactionSummary"
+import { TwoColumnLayout } from "Apps/Order/Components/TwoColumnLayout"
+import { track } from "Artsy/Analytics"
+import { ContextConsumer, Mediator } from "Artsy/SystemContext"
+import { Input } from "Components/Input"
+import { ErrorModal } from "Components/Modal/ErrorModal"
+import { Router } from "found"
+import React, { Component } from "react"
+import { createFragmentContainer, graphql, RelayProp } from "react-relay"
+import { Col, Row } from "Styleguide/Elements/Grid"
+import { HorizontalPadding } from "Styleguide/Utils/HorizontalPadding"
+import { get } from "Utils/get"
+import { Responsive } from "Utils/Responsive"
+import { OrderStepper } from "../../Components/OrderStepper"
+
+export interface OfferProps {
+  order: Offer_order
+  mediator: Mediator
+  relay?: RelayProp
+  router: Router
+}
+
+export interface OfferState {
+  offerValue: number | null
+  isCommittingMutation: boolean
+  isErrorModalOpen: boolean
+  errorModalTitle: string
+  errorModalMessage: string
+}
+
+function formatMoney(money: number | undefined): string | null {
+  if (money == null) return null
+
+  const [dollars, cents] = money.toFixed(2).split(".")
+  const dollarDigits = dollars.split("")
+
+  for (let i = dollarDigits.length - 3; i > 0; i -= 3) {
+    dollarDigits.splice(i, 0, ",")
+  }
+
+  const formattedDollars = dollarDigits.join("")
+
+  if (cents === "00") {
+    return `$${formattedDollars}`
+  }
+  return `$${formattedDollars}.${cents}`
+}
+
+@track()
+export class OfferRoute extends Component<OfferProps, OfferState> {
+  state = {
+    offerValue: null,
+    isCommittingMutation: false,
+    isErrorModalOpen: false,
+    errorModalTitle: null,
+    errorModalMessage: null,
+  }
+
+  componentDidMount() {
+    this.props.mediator.trigger("order:offer")
+  }
+
+  onContinueButtonPressed: () => void = () => {
+    this.setState({ isCommittingMutation: true }, () => {
+      if (this.props.relay && this.props.relay.environment) {
+        // TODO: commit mutation
+        new Promise(resolve => setTimeout(resolve, 1000)).then(() => {
+          this.setState({ isCommittingMutation: false })
+          this.setState({
+            isErrorModalOpen: true,
+            errorModalTitle: "Congratulations",
```

Comment:
hehe nice. To make sure all the code here is used, maybe we can call `onMutationError` here